cmd/kubeadm/app/apis/kubeadm/v1beta2/defaults.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
```

Comment:
the public types cannot include the internal types. this is breaking an api-machinery contract.